umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/echo10.clj

```diff
@@ -177,17 +177,30 @@
           :Explanation (when explanation
                          explanation)})))))
 
-(defn- parse-archive-dist-info
-  "Parses ArchiveAndDistributionInformation out of Echo 10 XML into UMM-C"
+(defn parse-and-set-archive-dist-info
+  "Parses price and data formats out of Echo 10 XML into UMM-C. The price
+   is the same for all formats.  There is no way to distinguish them in
+   ECHO 10."
   [doc]
   (let [price (value-of doc "Collection/Price")
-        format (value-of doc "Collection/DataFormat")]
-    (when (or price
-              format)
-      {:FileDistributionInformation [{:Fees price
-                                      :Format (or format
-                                                  u/not-provided)
-                                      :FormatType "Native"}]})))
+        formats (select doc "Collection/DataFormat")]
+    (if (and price (not formats))
```

Comment:
We need ```(seq (not formats))``` here I think

Reply:
If there are no formats then using (seq (not formats) gives me a null pointer exception.  The code works as is correctly without formats and with formats.  I test for this in my unit tests.

Reply:
If there are no formats then using (seq (not formats) gives me a null pointer exception.  The code works as is correctly without formats and with formats.  I test for this in my unit tests.